server/test/instant/db/instaql_test.clj

```diff
@@ -60,6 +62,114 @@
      (when (seq aggregates)
        {:aggregate aggregates}))))
 
+;; Snapshot update helpers for is-pretty-eq?
+;; Usage:
+;;  1. Write your test with a placeholder for the test value of `is-pretty-eq?`
+;;      (deftest my-new-test
+;;        (is-pretty-eq? (query-pretty {:users {}})
+;;                       '()))
+;;  2. Run `(update-snapshot #'my-new-test)` and the result will magically
+;;     replace '() with the result of `(query-pretty {:users {}})
+
+(defn find-node-by-location [zloc {:keys [line column]}]
+  (first
+   (z/find zloc z/next (fn [x]
+                         (let [{:keys [row col]} (meta (z/node x))]
+                           (when (and (= row line)
+                                      (= col column))
+                             (println (meta (z/node x))))
+                           (and (= row line)
+                                (= col column)))))))
+
+(defn- query-pretty [ctx r q]
+  (->> q
+       (iq/query ctx)
+       (resolvers/walk-friendly r)
+       (map ->pretty-node)))
+
+(defn- validation-err [ctx q]
+  (try (iq/query
+        ctx
+        q)
+       (catch clojure.lang.ExceptionInfo e
+         (-> (ex-data e)
+             ::ex/hint
+             :errors
+             first))))
+
+(defn pad-block [offset s]
+  (let [pad (apply str (repeat offset " "))]
+    (->> s
+         string/split-lines
+         ;; XXX: Need to skip first line?
```

Comment:
nit: is this comment intended?

Reply:
Turns out I didn't need to skip! Will remove.